helm/defectdojo/values.yaml

```diff
@@ -261,8 +261,12 @@ rabbitmq:
     erlangCookie: ""
     existingPasswordSecret: defectdojo-rabbitmq-specific
     existingErlangSecret: defectdojo-rabbitmq-specific
-    affinity: {}
-    nodeSelector: {}
+  memoryHighWatermark:
+    enabled: true
+    type: relative
+    value: 0.5
+  affinity: {}
```

Comment:
Was it you intention to change the indent of affinity and nodeSelector?

Reply:
@madchap  @alles-klar merged it as it was approved and only saw this afterwards

Reply:
It was intentional, yes. The `affinity` and `nodeSelector` are for the pods, and they were wrongly indented under the `auth` key.